contents/docs/libraries/dotnet/index.mdx

```diff
@@ -0,0 +1,153 @@
+```
+
+We strongly recommend reading our docs on [alias](/docs/product-analytics/identify#alias-assigning-multiple-distinct-ids-to-the-same-user) to best understand how to correctly use this method.
+
+## Group analytics
+
+Group analytics allows you to associate an event with a group (e.g. teams, organizations, etc.). Read the [Group Analytics](/docs/user-guides/group-analytics) guide for more information.
+
+> **Note:** This is a paid feature and is not available on the open-source or free cloud plan. Learn more on our [pricing page](/pricing).
+
+-   Capture an event and associate it with a group
+
+```csharp
+postHogClient.Capture(
+    "user_distinct_id", 
+    "some_event", 
+    groups: [new Group("company", "company_id_in_your_db")]);
+```
+
+-   Update properties on a group
+
+```csharp
+await postHogClient.GroupIdentifyAsync(
+    type: "company",
+    key: "company_id_in_your_db",
+    name: "Awesome Inc.",
+    properties: new()
+    {
+        ["employees"] = 11
+    }
+);
+```
+
+The `name` is a special property which is used in the PostHog UI for the name of the group. If you don't specify a `name` property, the group ID will be used instead.
+
+## Feature flags
+
+import FeatureFlagsLibsIntro from "../_snippets/feature-flags-libs-intro.mdx"
+
+<FeatureFlagsLibsIntro />
+
+import DotNetFeatureFlagsCode from '../../integrate/feature-flags-code/_snippets/feature-flags-code-dotnet.mdx'
+
+<DotNetFeatureFlagsCode />
+
+### Local evaluation
+
+import LocalEvaluationIntro from "../../feature-flags/snippets/local-evaluation-intro.mdx"
+
+<LocalEvaluationIntro />
+
+For details on how to implement local evaluation, see our [local evaluation guide](/docs/feature-flags/local-evaluation).
+
+## Experiments (A/B tests)
+
+Since [experiments](/docs/experiments/manual) use feature flags, the code for running an experiment is very similar to the feature flags code:
+
+```csharp
+if (await posthog.GetFeatureFlagAsync(
+        "experiment-feature-flag-key",
+        "user_distinct_id")
+    is { VariantKey: "variant-name" })
+{
+    // Do something
+}
+```
+
+It's also possible to [run experiments without using feature flags](/docs/experiments/running-experiments-without-feature-flags). 
+
+
+## GeoIP properties
+
+The posthog-dotnet library disregards the server IP, does not add the GeoIP properties, and does not use the values for feature flag evaluations.
+
+## Serverless environments (Azure Functions/Render/Lambda/...)
+
+By default, the library buffers events before sending them to the `/batch` endpoint, for better performance. This can
+lead to lost events in serverless environments, if the .NET process is terminated by the platform before the buffer
+is fully flushed. To avoid this, you can:
+
+- ensure that `await posthog.FlushAsync()` is called after processing every request, by adding a middleware to your server: this allows `posthog.Capture()` to remain asynchronous for better performance.
```

Comment:
```suggestion
By default, the library buffers events before sending them to the `/batch` endpoint for better performance. This can lead to lost events in serverless environments if the .NET process is terminated by the platform before the buffer is fully flushed. 

To avoid this, ensure that `await posthog.FlushAsync()` is called after processing every request, by adding a middleware to your server. This allows `posthog.Capture()` to remain asynchronous for better performance.
```